src/main/jbake/content/using-git.ad

```diff
@@ -0,0 +1,178 @@
+GitHub repo or to Git at Apache (i.e. git-wip-us.apache.org). You will want to fork GitHub's apache/opennlp to your own
+account on GitHub, this will enable Pull Requests of your own. Cloning this fork locally will set up "origin" to point
+to your remote fork on GitHub as the default remote. So if you perform "git push origin master" it will go to GitHub.
+
+To attach to the apache Git repo do the following:
+
+    git remote add apache https://git-wip-us.apache.org/repos/asf/opennlp.git
+
+To check your remote setup:
+
+    git remote -v
+
+You should see something like this:
+
+    origin    https://github.com/your-github-id/opennlp.git (fetch)
+    origin    https://github.com/your-github-id/opennlp.git (push)
+    apache    https://git-wip-us.apache.org/repos/asf/opennlp.git (fetch)
+    apache    https://git-wip-us.apache.org/repos/asf/opennlp.git (push)
+
+Now if you want to experiment with a branch, this by default points to your GitHub account because "origin" is default.
+You can work as you normally do using just GitHub, until you are ready to merge with the Apache remote repository.
+Some conventions will integrate with Apache JIRA ticket numbers.
+
+    git checkout -b opennlp-xxxx #xxxx typically is a JIRA ticket number
+    #do some work on the branch
+    git commit -a -m "doing some work"
+    git push origin opennlp-xxxx # notice pushing to **origin** not **apache**
+
+Once you are ready to commit to the Apache remote you can merge and push them directly, or better yet create a
+pull request (PR). 
+
+## How to create a PR (committers)
+
+Push your branch to GitHub:
+
+    git checkout opennlp-xxxx
+    git push origin opennlp-xxxx
+
+Go to your opennlp-xxxx branch on GitHub. Since you forked it from GitHub's apache/opennlp it will default any PR to
+go to apache/master. 
+
+* Click the green "Compare, review, and create pull request" button. 
+* You can edit the _to_ and _from_ for the PR if it is not correct. The "base fork" should be apache/opennlp unless
+you are collaborating separately with one of the committers on the list. The "base" will be master. Do not submit a
+PR to one of the other branches unless you know what you are doing. The "head fork" will be your forked repo and the
+"compare" will be your opennlp-xxxx branch. 
+* Click the "Create pull request" button and name the request "OPENNLP-XXXX" (uppercase). This will connect the
+comments of the PR to the mailing list and JIRA comments.
+* From now on the PR lives on GitHub's apache/opennlp. You can use the commenting UI there. 
+* If you are looking for a review or sharing with someone else say so in the comments but do not worry about 
+automated merging of your PR -- you will have to do that later. The PR is tied to your branch so you can respond to 
+comments, fix code, and commit from your local repository. They will appear on the PR page and be mirrored to JIRA 
```

Comment:
We should suggest to use git commit --amend and git push -f to to react on feedback